gradle.properties

```diff
@@ -1,9 +1,9 @@
 org.gradle.caching=true
 group=org.ballerinalang
 # During the release workflow, the following will get bumped automatically
-version=2201.7.1-SNAPSHOT
+version=2201.7.2-SNAPSHOT
```

Comment:
Need to update the ballerina-test-automation file as well https://github.com/ballerina-platform/ballerina-distribution/blob/31092089977b612c24b8206417833c0fe215afd2/ballerina-test-automation/gradle.properties#L1